airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/AbstractDbSource.java

```diff
@@ -67,6 +90,9 @@
 public abstract class AbstractDbSource<DataType, Database extends AbstractDatabase> extends
     BaseConnector implements Source, AutoCloseable {
 
+  protected static final Set<JDBCType> allowedCursorTypes = Set.of(TIMESTAMP, TIME, DATE, BIT, BOOLEAN, TINYINT, SMALLINT, INTEGER,
```

Comment:
BIT, BOOLEAN, BINARY, BLOB - not sure that we can use that.

Reply:
the same issue is here,
https://github.com/airbytehq/airbyte/blob/83a89aa843e587a02815eabb8c17437c0b8339e8/airbyte-db/db-lib/src/main/java/io/airbyte/db/jdbc/JdbcSourceOperations.java#L64